### PR TITLE
Add recommended new player guides to main page

### LINKED ISSUE #490: We don't have a curated list of recommended guides front-and-center on the wiki
### Problem

New players are introduced to the wiki during the splash screen now, and will be looking for new-player guides that get them on their feet as quickly as possible. They may not be willing to trawl through everything on the top bar looking for things they need early-game.

### Context

The index page should have a list of curated, helpful guides for new players that does not require any complex wiki navigation to find. These should target early-game content ('what is marks?', 'what do I do when I am done with marks?', etc)

### PR DESCRIPTION
**Pull request by matr1x-hackmud**
### Problem

Fixes #490

### CODE REVIEW

File: docs/index.mdx [hunk 1]
```diff
@@ -11,6 +11,11 @@ Welcome to the hackmud wiki!
 
 Use the nav bar at the top of the page to browse to the content you are looking for.
 
+#### Recommended new player guides
+
+- [[marks:Intro to the Marks System]]
```
Comment (seanmakesgames) [inline docs/index.mdx:16]: this link is not working correctly

Comment (seanmakesgames) [reply docs/index.mdx:16]: (tested via codespace)

Comment (matr1x-hackmud) [reply docs/index.mdx:16]: No links are - #489

Comment (matr1x-hackmud) [reply docs/index.mdx:16]: In my codespace, both seem to be loading fine:
image

**Review by seanmakesgames — APPROVED** [review]

**Pull request merged**